psalm.xml

```diff
@@ -56,7 +40,16 @@
         <MethodSignatureMismatch>
             <errorLevel type="suppress">
                 <file name="src/Behat/Testwork/EventDispatcher/TestworkEventDispatcherSymfonyLegacy.php" />
+                <file name="src/Behat/Testwork/EventDispatcher/TestworkEventDispatcherSymfony5.php" />
```

Comment:
The new PHPStan also finds this problem in this file

Reply:
This is not really related to new Psalm. It is related to the fact that you removed the part forcing to use Symfony 5 in the static analysis job

Reply:
Ah, yes, you are right. Nevertheless, we need to add this to be able to run it with other Symfony versions